Sources/Plot/API/Country.swift

```diff
@@ -0,0 +1,260 @@
+/**
+*  Plot
+*  Copyright (c) John Sundell 2019
+*  MIT license, see LICENSE file for details
+*/
+
+// Enum defining country codes according to the ISO 3166-1 standard,
+// which can be used to specify the localized language of a page with
+// <link rel="alternate" href="http://fqdn.site/" hreflang="en-GB"> 
+public enum Country: String {
```

Comment:
The `Language` enum already includes some variants, like `.ukEnglish` and `.usEnglish`. Is this really needed, given that it lets you form invalid combinations, such as `en-SV`? I think extending `Language` with variants as needed is a better approach.

Reply:
ok, I must admit here that I am probably not the best person to architect the localization as I have very little experience with languages other than English, specifically the US variation.  And I completely overlooked those 2 entries in Language.swift, so I removed the attempt at ISO 3166, and added caEnglish "en-ca", and auEnglish "en-au".